packages/lodestar/src/chain/chain.ts

```diff
@@ -338,7 +329,7 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
     await this.db.setChainHeadRoots(blockRoot, block.stateRoot);
```

Comment:
I think this line should be removed, since we'll be doing it in applyForkChoiceRule if needed